.github/workflows/test.yml

```diff
@@ -33,13 +33,40 @@ jobs:
           # Dec 12, 2021
           git checkout 74278b9b7cf816f0356181f387012fdeb6d65b52
           autoreconf -fis
+          # compile for x86_64
           ./configure --prefix=/opt/vde
           make
           sudo make install
+          # cleanup
+          make distclean
+          # cross-compile for arm64
+          export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
+          export CC=$(xcrun --sdk macosx --find clang)
+          export CXX=$(xcrun --sdk macosx --find clang++)
+          export CFLAGS="-arch arm64e -isysroot $SDKROOT -Wno-error=implicit-function-declaration"
+          export CXXFLAGS=$CFLAGS
+          ./configure --prefix=/opt/vde.arm64 --host=arm-apple-darwin --target=arm-apple-darwin --build=x86_64-apple-darwin
+          make
+          sudo make install
+          unset SDKROOT CC CXX CFLAGS CXXFLAGS
       - name: Make
-        run: make PREFIX=/opt/vde
+        run: |
+          # for x86_64
+          make PREFIX=/opt/vde
+          # for arm64
+          export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
+          export CC=$(xcrun --sdk macosx --find clang)
+          export CXX=$(xcrun --sdk macosx --find clang++)
+          export CFLAGS="-arch arm64e -isysroot $SDKROOT -Wno-error=implicit-function-declaration"
+          export CXXFLAGS=$CFLAGS
+          make PREFIX=/opt/vde.arm64
+          unset SDKROOT CC CXX CFLAGS CXXFLAGS
       - name: Install
-        run: sudo make PREFIX=/opt/vde install
+        run: |
+          # for x86_64
+          sudo make PREFIX=/opt/vde install
+          # for arm64
+          sudo make PREFIX=/opt/vde.arm64 install
```

Comment:
This doesn't seem to install arm64 binary.

Probably you should install the binaries in `name: Make` section

Reply:
Thanks for the comment. 

[This](https://github.com/lima-vm/vde_vmnet/pull/33/commits/4ffc5f735def87efda3a3f1ff2f0db1de74c4d17) should fix the issue, I assume.